compiler/src/iree/compiler/Dialect/Flow/Conversion/TensorToFlow/Patterns.cpp

```diff
@@ -20,6 +20,29 @@ namespace mlir::iree_compiler::IREE::Flow {
 
 namespace {
 
+/// Converts linalg.fill ops into flow.tensor.splat ops.
+///
+/// This is expected to improve performance because we can use DMA
+/// functionalities for the fill, instead of dispatching kernels.
+struct ConvertLinalgFillPattern final
+    : public OpRewritePattern<linalg::FillOp> {
+  using OpRewritePattern::OpRewritePattern;
+
+  LogicalResult matchAndRewrite(linalg::FillOp fillOp,
+                                PatternRewriter &rewriter) const override {
+    if (fillOp->getParentOfType<IREE::Flow::DispatchWorkgroupsOp>()) {
```

Comment:
I have been using this https://github.com/MaheshRavishankar/iree/blob/59f93aafc6a9465f7f3234dcdad76c8812d2f898/compiler/src/iree/compiler/Dialect/Flow/Transforms/RegionOpUtils.cpp#L122 . But this is fine.